# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/strace/arch-alpha-bjm.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./bjm.c.orig	Sun Oct  8 00:15:38 2000
+++ ./bjm.c	Sun Oct  8 00:16:03 2000
@@ -33,6 +33,9 @@
 
 #if defined(LINUX)
 
+/* On Alpha system this header file produces a compiler error */
+#define _ALPHA_CURRENT_H   // disable asm/current.h [2.4.0-test9]
+
 #include <fcntl.h>
 #include <sys/stat.h>
 #include <sys/time.h>
